### DISCUSSION TITLE
Getting error "mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64')" in MacOXS when running Mike

### DISCUSSION THREAD

**Discussion by sunilkn020** [jimporter/mike#195] [General]
Hi All,
I'm getting the below error when I try to run "Mike deploy" command. Kindly suggest how I can resolve it.
"error: dlopen(/Users/sunilkn/Library/Python/3.10/lib/python/site-packages/regex/_regex.cpython-310-darwin.so, 0x0002): tried: '/Users/sunilkn/Library/Python/3.10/lib/python/site-packages/regex/_regex.cpython-310-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64')), '/System/Volumes/Preboot/Cryptexes/OS/Users/sunilkn/Library/Python/3.10/lib/python/site-packages/regex/_regex.cpython-310-darwin.so' (no such file), '/Users/sunilkn/Library/Python/3.10/lib/python/site-packages/regex/_regex.cpython-310-darwin.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64'))"

Below is the pip list of my Mac os:
Babel                                     2.14.0
bracex                                    2.2.1
certifi                                   2023.11.17
charset-normalizer                        3.3.2
click                                     8.1.7
colorama                                  0.4.6
ghp-import                                2.1.0
gitdb                                     4.0.9
GitPython                                 3.1.27
idna                                      3.6
importlib-metadata                        4.11.3
importlib-resources                       6.1.1
Jinja2                                    3.1.3
Markdown                                  3.5.2
MarkupSafe                                2.1.3
mergedeep                                 1.3.4
mike                                      2.0.0
mkdocs                                    1.5.3
mkdocs-awesome-pages-plugin               2.7.0
mkdocs-git-revision-date-localized-plugin 1.0.1
mkdocs-macros-plugin                      0.7.0
mkdocs-material                           9.5.4
mkdocs-material-extensions                1.3.1
packaging                                 23.2
paginate                                  0.5.6
pathspec                                  0.12.1
pip                                       23.3.2
platformdirs                              4.1.0
Pygments                                  2.17.2
pymdown-extensions                        10.7
pyparsing                                 3.0.7
python-dateutil                           2.8.2
pytz                                      2022.1
PyYAML                                    6.0.1
pyyaml_env_tag                            0.1
regex                                     2023.12.25
requests                                  2.31.0
setuptools                                58.1.0
six                                       1.16.0
smmap                                     5.0.0
termcolor                                 2.1.1
urllib3                                   2.1.0
verspec                                   0.1.0
watchdog                                  3.0.0
wcmatch                                   8.3
wheel                                     0.37.1
zipp                                      3.8.0

**Comment by jimporter** [comment]
This looks like an issue with your Python installation, not with mike; mike doesn't contain any natively-compiled code. You should probably reinstall Python on your system.
